plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/server/ServerSettingsWizardPage.java

```diff
@@ -497,9 +495,9 @@ private SelectionListener onImportProject(ServerSettingsWizardPageModel model, f
             @Override
             public void widgetSelected(SelectionEvent e) {
                 Map<com.openshift.restclient.model.IProject, Collection<IBuildConfig>> projectsAndBuildConfigs = new HashMap<>();
-                List<IBuildConfig> buildConfigs = new ArrayList<>();
-                ResourceUtils.getBuildConfigFor(model.getResource(), buildConfigs);
-                projectsAndBuildConfigs.put(model.getResource().getProject(), buildConfigs);
+                if (model.getResource() != null) {
+                	projectsAndBuildConfigs.put(model.getResource().getProject(), Collections.emptyList());
+                }
```

Comment:
the list of build configs is retrieved in `ImportApplicationWizard`
Look into lines 501-502 , before my changes - they do nothing. `ResourceUtils.getBuildConfigFor` was called and it is getter - so it returns some value. But why isn't it assigned to any variable and used after? This code is useless. Look inside `ResourceUtils.getBuildConfigFor` and examples of its usage in other places and you'll get the idea.
So my fix just removes the NPE indeed and nothing else. As you said, test it and you'll see that it works.